usr/local/www/diag_dns.php

```diff
@@ -29,20 +29,24 @@
 */
 
 /*
-	pfSense_MODULE:	dns
+	pfSense_MODULE: dns
 */
 
-$pgtitle = array(gettext("Diagnostics"),gettext("DNS Lookup"));
+/* Bootstrap conversion: sbeaver@netgate.com */
+
+$pgtitle = array(gettext("Diagnostics"),gettext("DNS lookup"));
 require("guiconfig.inc");
 
 $host = trim($_REQUEST['host'], " \t\n\r\0\x0B[];\"'");
 $host_esc = escapeshellarg($host);
 
-if (is_array($config['aliases']['alias'])) {
-	$a_aliases = &$config['aliases']['alias'];
-} else {
-	$a_aliases = array();
+/* If this section of config.xml has not been populated yet we need to set it up 
+*/
+if (!is_array($config['aliases']['alias'])) {
```

Comment:
I can do that, but I am concerned that the page does not work as required without this bug-fix and so will not pass unit testing.

Steve
--
Stephen Beaver | +1 (512) 646-4100;239 | sbeaver@netgate.com | www.netgate.com 

> On Mar 28, 2015, at 7:54 AM, SjonHortensius <notifications@github.com> wrote:
> 
> In usr/local/www/diag_dns.php <https://github.com/SjonHortensius/pfsense/pull/42#discussion_r27344508>:
> 
> >  require("guiconfig.inc");
> >  
> >  $host = trim($_REQUEST['host'], " \t\n\r\0\x0B[];\"'");
> >  $host_esc = escapeshellarg($host);
> >  
> > -if (is_array($config['aliases']['alias'])) {
> > -	$a_aliases = &$config['aliases']['alias'];
> > -} else {
> > -	$a_aliases = array();
> > +/* If this section of config.xml has not been populated yet we need to set it up 
> > +*/
> > +if (!is_array($config['aliases']['alias'])) {
> Then please, revert it. Base your changes on this repo and let me merge from upstream now and then. It'll keep things clear. I only want bootstrap related changes in this repo :)
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/SjonHortensius/pfsense/pull/42/files#r27344508>.
>